run-email.sh

```diff
@@ -0,0 +1 @@
+MONGO_URL=mongodb://localhost:27017/wekan MAIL_URL=smtp://noreplytest:noreply110@smtp.126.com:25/ MAIL_FROM=noreplytest@126.com meteor
```

Comment:
I think this shouldn't be here... And I think you should change your password **now**.